src/components/AppSettings.vue

```diff
@@ -237,6 +237,8 @@ export default {
             return state.settings;
         },
         settingShowAutoComplete: bindSetting('showAutocomplete'),
+        settingAutoCompleteDesc: bindSetting('autoCompleteDesc'),
```

Comment:
These appear to be unused

Reply:
These settings are used :) - they are enabled and found in the advanced settings section of Kiwi